ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallDialog.kt

```diff
@@ -50,15 +50,19 @@ fun PaywallDialog(
         val dismissRequest = {
             shouldDisplayDialog = false
         }
+        val paywallOptions = paywallDialogOptions.toPaywallOptions(dismissRequest)
+
+        val viewModel = getPaywallViewModel(options = paywallOptions)
```

Comment:
I needed the viewmodel in order to track this event when dismissing the dialog correctly. It shouldn't cause duplications since we don't have keys anymore and it's the same model.